launcher/experiments/experiments.go

```diff
@@ -0,0 +1,78 @@
+// Package experiments contains functionalities to retrieve syned experiments
```

Comment:
"Package experiments contains functionalities to retrieve synced experiments"

Reply:
Done